seed/docs/views.py

```diff
@@ -0,0 +1,65 @@
+# !/usr/bin/env python
+# encoding: utf-8
+"""
+:copyright (c) 2014 - 2020, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.  # NOQA
+:author
+"""
+from django.shortcuts import render
+
+from collections import namedtuple
+import os
+import re
+
+import markdown
+import yaml
+
+
+YAML_DOC_BOUNDARY = re.compile(r"^-{3,}\s*$", re.MULTILINE)
```

Comment:
Would there be any issue with parsing multiple `-`'s used for title headers in the markdown section?

Reply:
Shouldn't be a problem, the way we use `split()` just splits on the first two occurrences:
https://github.com/SEED-platform/seed/blob/109d509c5131eac823f8f64921ce7e6bcdc3bb70/seed/docs/views.py#L40

ref: https://docs.python.org/3/library/re.html#re.split